### ISSUE TITLE
About model performance after training 

### ISSUE THREAD

**Issue by lucaskyle** [lee-man/movenet#22]
i modified some lines
now it can load lightning movenet model 
i try your training code only using COCO dataset 

training sample : 64115
validation sample : 5000

when i try finetuning published lightning movenet, the loss will be like:
--->train|loss 4.476185 | hm_loss 1.625346 | hp_loss 1.318379 | hm_hp_loss 1.305676 | hp_offset_loss 0.226784 |​
--->valid| loss 3.907756 | hm_loss 1.076794 | hp_loss 0.682829 | hm_hp_loss 2.044044 | hp_offset_loss 0.104088 |

when i train the lightning movenet from scratch,  the loss will be like:
--->train|loss 8.478377 | hm_loss 2.583637 | hp_loss 2.864822 | hm_hp_loss 2.782381 | hp_offset_loss 0.247538 | ​
--->valid| loss 7.517864 | hm_loss 1.706246 | hp_loss 1.409458 | hm_hp_loss 4.288480 | hp_offset_loss 0.113680 |​
is it good?​

then i try show demo results with both models,
the model trained from scratch can not even detect key points.

**Comment by lee-man** [comment]
hi, @lucaskyle 
This repo is not longer maintained. Sorry that I cannot offer valuable feedback to you.

**Comment by lucaskyle** [comment]
@lee-man 
understood, thank u
